presto/middleware.nim

```diff
@@ -0,0 +1,33 @@
+#
+#         REST API framework implementation
+#             (c) Copyright 2021-Present
+#         Status Research & Development GmbH
+#
+#              Licensed under either of
+#  Apache License, version 2.0, (LICENSE-APACHEv2)
+#              MIT license (LICENSE-MIT)
+
+{.push raises: [].}
+
+import chronos, chronos/apps/http/httpserver
+import route, servercommon, serverprivate
```

Comment:
`"."/[route, ...]` avoids the risk of loading these modules from a different package

Reply:
Fixed in https://github.com/status-im/nim-presto/pull/73/commits/0fc1d8dbdec54b75deff52789042ec63f3ceaf3d